.vscode/launch.json

```diff
@@ -20,7 +24,19 @@
       "name": "Attach to Server",
       "port": 6009,
       "restart": true,
-      "outFiles": ["${workspaceRoot}/dist/server/*.js"]
+      "outFiles": [
+        "${workspaceRoot}/dist/server/*.js"
+      ]
+    },
+    {
+      "type": "node",
+      "request": "attach",
+      "name": "Attach to Jasmine Client",
```

Comment:
I actually made this for debugging the server and client tests though I suppose it would also work for the LSP tests. It's simply set up to attach to the default port used by jasmine.